Build/packages.config

```diff
@@ -1,6 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Remotion.BuildScript" version="2.0.0-alpha022" targetFramework="net45" developmentDependency="true" />
+  <package id="DependDB.BuildProcessor" version="2.0.0" targetFramework="net45" />
+  <package id="DependDB.BuildProcessor.NuGetPreProcessor" version="2.0.0" targetFramework="net45" />
+  <package id="JetBrains.Annotations" version="2019.1.3" targetFramework="net45" />
+  <package id="MSBuild.Extension.Pack" version="1.5.0" targetFramework="net45" />
+  <package id="NUnit.ConsoleRunner" version="3.11.1" targetFramework="net45" />
+  <package id="NUnit.Extension.NUnitV2Driver" version="3.8.0" targetFramework="net45" />
+  <package id="NuGet.CommandLine" version="3.5.0" targetFramework="net45" developmentDependency="true" />
   <package id="NuGet.for.MSBuild" version="1.4.3" targetFramework="net45" developmentDependency="true" />
-  <package id="Remotion.BuildScript" version="2.0.0-alpha018" targetFramework="net45" developmentDependency="true" />
+  <package id="Remotion.BuildTools.MSBuildTasks" version="1.0.6862.28795" targetFramework="net45" developmentDependency="true" />
```

Comment:
update in .nuget/packages.config is missing. See also https://github.com/re-motion/Relinq-SqlBackend/pull/4/files#r422187249